js/lbank2.js

```diff
@@ -258,6 +258,9 @@ module.exports = class lbank2 extends Exchange {
                     'btctron': 'BTCTRON',
                     'xrp': 'XRP',
                 },
+                'defaultNetworks': {
+                    'USDT': 'TRC20',
```

Comment:
we usually default to `ERC20` no ?

Reply:
> we usually default to `ERC20` no ?
In the past, when I asked this question to Igor, he said that ETH was redundant for USDT. To remind, we have two things "defaultNetwork" and "defaultNetworks"(plural). ETH might be candidate of "defaultNetwork" (so, all coins, which are not specifically declared in "defaultNetworks", would use from "defaultNetwork"), however, for USDT, most ppl use TRC20.
Also, I think we might make `networkCode` as mandatory argument, to avoid any such "defaults" at all (which I think will be much better).